kotlinx-coroutines-core/common/src/internal/Symbol.kt

```diff
@@ -10,7 +10,7 @@ package kotlinx.coroutines.internal
  * @suppress **This is unstable API and it is subject to change.**
  */
 internal class Symbol(val symbol: String) {
-    override fun toString(): String = symbol
+    override fun toString(): String = "<$symbol>"
```

Comment:
Why this is needed?

Reply:
This is just more readable, to distinguish symbols from strings/enums